### DISCUSSION TITLE
the application does not open

### DISCUSSION THREAD

**Discussion by jhonkyke** [pear-devs/pear-desktop#2018] [Q&A] [answered]
I installed Ubuntu 24.04 lts and since then the app does not open, I tried to uninstall and reinstall but the problem persists, I tried to install the snap package but it does not even allow it to be installed.
Has this happened to anyone else?

**Comment by fmendezy** [comment]
i confirm, in ubuntu 24.04 LTS this app dont open.

**Comment by JellyBrick** [comment, marked as answer]
See #2007 
@fmendezy @jhonkyke
